model/testing/src/icon4py/model/testing/pytest_config.py

```diff
@@ -123,6 +123,9 @@ def pytest_addoption(parser):
 
 
 def pytest_runtest_setup(item):
+    for marker_i in item.own_markers:
+        if marker_i.name == "embedded_remap_error" and item.config.getoption("--backend"):
```

Comment:
Shouldn't it be like this?
```suggestion
        if marker_i.name == "embedded_remap_error" and item.config.getoption("--backend" == "embedded"):
```